api/services/index.js

```diff
@@ -5,3 +5,5 @@ exports.AuthService = require('./AuthService');
 exports.UserService = require('./UserService');
 exports.CategoryService = require('./CategoryService');
 exports.QuestionService = require('./QuestionService');
+exports.SurveyResultService = require('./SurveyResultService');
```

Comment:
Make sure you have the most recent changes, this line already exists as well as another line.